src/capnp_compile.erl

```diff
@@ -323,37 +323,37 @@ generate_decode_fun(Line, TypeId, SortedDataFields, SortedPtrFields, Groups, Sch
 	end,
 	Name = decoder_name(TypeId, Schema),
 	ast_function(
-		{'$uberpt_quote', Name},
+		quote(Name),
 		fun
-			({'$uberpt_quote', DataMatcher}, {'$uberpt_quote', PointerMatcher}, {'$uberpt_quote', MessageRefVar}) ->
+			(quote(DataMatcher), quote(PointerMatcher), quote(MessageRefVar)) ->
```

Comment:
This is the thing that was breaking on 21?

Reply:
Ooops, off by one error.